src/components/CardContainer/CardContainer.js

```diff
@@ -30,12 +30,28 @@ class CardContainer extends React.Component {
             .then(res => {
                 let random = UniqueRandomArray(res)
                 let dogs = [random(), random()]
-                const state = {...this.state, dogs: dogs}
+                const state = {...this.state, dogs: dogs, randomDog: random}
                 this.setState(state)
                 }
             )
     }
 
+    clickUpdateWin = (id) => {
+        fetch('http://localhost:3000/dogs/' + id + "/win", {
```

Comment:
I've put a comment on the back-end about the API naming, might be worth checking what has been done there. At the moment this is fine though